srcpkgs/persepolis/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'persepolis'
+pkgname=persepolis
+version=3.2.0
+revision=1
+archs="noarch"
+build_style=python3-module
+hostmakedepends="python3 python3-setuptools"
+depends="python3-setproctitle python3-requests python3-psutil aria2 ffmpeg
+	libnotify pulseaudio youtube-dl sound-theme-freedesktop qt5-svg"
+short_desc="Aria2 Python GUI"
+maintainer="Nathan Owens <ndowens04@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://persepolisdm.github.io/"
+distfiles="https://github.com/persepolisdm/${pkgname}/archive/${version}.tar.gz"
+checksum=d27cf2a4e02b0fbe79b1903ca0ab32a6007493d51a8201443a8febb566750acd
+
+pre_build() {
+# Remove input, otherwise will stop install
+sed -e '/input/d' \
+	-e '/if answer not/d' \
+	-e '/sys.exit(1)/d' -i setup.py
+}
```

Comment:
please make this a patch, or at least put it in `do_patch()` and describe why this is necessary

Reply:
Done. I did this as it reduces the amount of dependencies during build, when som are only needed during runtime, as it will check for those too and stop the build if the dependencies are not found